depth_image_topic: /camera/depth/image
camera_info_topic: /camera/depth/camera_info
downsampling_factor: 8

resolution:
  width: 640
  height: 480

